stan/math/rev/core/init_chainablestack.hpp

```diff
@@ -20,8 +20,9 @@ namespace math {
  * hook ensures that each worker thread has an initialized AD tape
  * ready for use.
  *
- * Refer to https://software.intel.com/en-us/node/506314 for details
- * on the observer concept.
+ * Refer to
+ * https://software.intel.com/content/www/us/en/develop/documentation/tbb-documentation/top/intel-threading-building-blocks-developer-reference/task-scheduler/taskschedulerobserver.html
+ * for details on the observer concept.
```

Comment:
I think we should put links like this through wayback machine so we can capture them. I've had instances of linked docs no longer working

https://web.archive.org/web/20210323211920/https://software.intel.com/content/www/us/en/develop/documentation/tbb-documentation/top/intel-threading-building-blocks-developer-reference/task-scheduler/taskschedulerobserver.html